docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     ports:
       - "${EXPOSE_SENTRY_PORT:-8090}:8090"
     command: run --host 0.0.0.0 --port 8090
+    stop_signal: SIGINT
```

Comment:
If we're going to do this, we should update `docker/images/fakesentry/Dockerfile`, too. I think we can copy this:

https://github.com/mozilla-services/socorro/blob/main/docker/Dockerfile.fakesentry

Reply:
i think this PR already includes that change?

Reply:
Whoops--you're right! Not sure how I missed it.